lib/still/preprocessor.ex

```diff
@@ -16,25 +16,28 @@ defmodule Still.Preprocessor do
     URLFingerprinting,
     Save,
     AddLayout,
-    AddContent
+    AddContent,
+    Image
   }
 
   @default_preprocessors %{
-    ".slim" => [AddContent, Frontmatter, Slime, OutputPath, AddLayout, Save],
-    ".slime" => [AddContent, Frontmatter, Slime, OutputPath, AddLayout, Save],
-    ".eex" => [AddContent, Frontmatter, EEx, OutputPath, AddLayout, Save],
+    ".slim" => [AddContent, EEx, Frontmatter, Slime, OutputPath, AddLayout, Save],
+    ".slime" => [AddContent, EEx, Frontmatter, Slime, OutputPath, AddLayout, Save],
+    ".eex" => [AddContent, EEx, Frontmatter, EEx, OutputPath, AddLayout, Save],
```

Comment:
Is this intentional? You have `EEx` twice

Reply:
No 👍